core/kazoo_services/src/kz_services_quantities.erl

```diff
@@ -532,8 +532,9 @@ calculate_qubicle_queue_updates(JObj, Updates) ->
     case kz_doc:type(JObj) =:= <<"qubicle_queue">> of
         'false' -> Updates;
         'true' ->
-            Key = [<<"qubicle">>, <<"queues">>],
-            [{Key, 1} | Updates]
+            BKey = [<<"qubicle">>, <<"queues">>],
+            PKey = [<<"qubicle">>, <<"pro_queues">>],
+            [{BKey, 1}, {PKey, 1} | Updates]
```

Comment:
@mark2600 umm? for any queue encountered this adds 1 to both the queues and pro_queues count.  Need a condition in there to choose one or the other based on the offering.  However, with the above suggestions you can simply this as well to...

Key = [<<"qubicle">>, <<(kz_json:get_ne_value(<<"pvt_qubicle_offering">>, JObj, <<"basic">>))/binary, "_queues">>],
[{Key, 1}, {PKey, 1} | Updates]